common/src/backend/resources/assets/flywheel/flywheel/internal/light_lut.glsl

```diff
@@ -44,9 +44,14 @@ bool _flw_nextLut(uint base, int coord, out uint next) {
     return false;
 }
 
-bool _flw_chunkCoordToSectionIndex(ivec3 sectionPos, out uint index) {
+bool _flw_chunkCoordToSectionIndex(uint sceneId, ivec3 sectionPos, out uint index) {
+    uint scene;
+    if (_flw_nextLut(0u, int(sceneId), scene) || scene == 0u) {
```

Comment:
The cast to int here is a little spooky since the data really is a uint, but we'll realistically run out of memory before this causes issues.

Still, it could be good to have a copy of _flw_nextLut that takes a uint index